src/stravalib/client.py

```diff
@@ -271,7 +271,7 @@ def get_activities(
         before: datetime | str | None = None,
         after: datetime | str | None = None,
         limit: int | None = None,
-    ) -> BatchedResultsIterator[model.Activity]:
+    ) -> BatchedResultsIterator[model.SummaryActivity]:
```

Comment:
For every endpoint, we should check the Strava API docs for the correct return type

Reply:
should this be a new issue @jsamoocha ?

Reply:
I added the list of all endpoints to your new issue.